acceptance-tests/src/test/java/uk/gov/di/test/step_definitions/Login.java

```diff
@@ -7,11 +7,7 @@
 import org.junit.jupiter.api.Assertions;
 import org.openqa.selenium.support.ui.ExpectedCondition;
 import org.openqa.selenium.support.ui.WebDriverWait;
-import uk.gov.di.test.pages.CheckYourEmailPage;
-import uk.gov.di.test.pages.EnterYourPasswordPage;
-import uk.gov.di.test.pages.LoginPage;
-import uk.gov.di.test.pages.ResetYourPasswordPage;
-import uk.gov.di.test.pages.TermsAndConditionsPage;
+import uk.gov.di.test.pages.*;
```

Comment:
need to turn off * imports in the ide

Reply:
Done. also updated other occurrences of this.